.github/workflows/ci-build.yml

```
@@ -1,6 +1,8 @@
 name: CI Build
 
-on: [push, pull_request]
+on:
+  pull_request:
+    branches: main
```

It's worth building PR's to any branch
```suggestion
on: [pull_request]
```